css/styles.css

```diff
@@ -0,0 +1,174 @@
+* { 
+  box-sizing: border-box; 
+}
+
+body {
+  background-color: none;
+  width: 960px;
+  height: 100vh;
+  font-family: "Source Sans Pro";
+  margin: 0px;
+}
+
+h1 {
+  color:yellow;
+  position: relative;
+  left: 3%;
+  top: 13%;
+}
+
+nav {
+  position: relative;
+  right: 45px;
+  top:5px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-content: space-between;
+}
+
+nav a {
+  margin-left: 15px;
+}
+
+span {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+u {
+  text-decoration-color: yellow;
+}
+
+a {
+  color:white;
+  text-decoration: none;
+}
+
+h3 {
+  font-weight: bolder;
+  font-size: 24px;
+}
+
+h6 {
+  color: white;
+  font-size: 18px;
+  position: relative;
+  left: 5%;
+  top: 35%;
+  margin: 10px, 10px, 10px, 10px;
+}
+
+.header {
+  background-color: black;
+  height: 15%;
+}
+
+.break {
+  height: 3px;
+  background-color: yellow;
+}
+.subtitle {
+  clear: left;
+  background-color: lightgray;
+  height: 50px;
+ }
+
+ .subtitle > p {
+  text-align: center;
+  margin: 0px;
+  position: relative;
+  top:15px;
+  font-size: 20px;
+ }
+
+.header {
+  background-color: black;
+}
+
+.image{
+  background-image: url('../images/coworking.jpg');
+  background-size: contain;
+  height: 50%;
+
+}
+
+.action{
+  background-color: hsla(0,0%,100%,0.8);
+  width: 35%;
+  height: 90%;
```

Comment:
@zdburnett remove the height here, and let the content dictate the containers height. Use margin and padding to adjust position to match the comp 🔨

Reply:
done.